packages/components/src/utils/validation/validatePropChange.spec.ts

```diff
@@ -0,0 +1,48 @@
+import { validatePropChange } from './validatePropChange';
+
+describe('validatePropChange()', () => {
+  const data: {
+    newVal: Record<string, any>;
+    oldVal: Record<string, any>;
+    propertyName: string;
```

Comment:
When one parameter is called `arrayOfRelevantPropNames` why isn't this one called `propName`?